packages/react-core/src/components/Dropdown/examples/Dropdown.md

```diff
@@ -20,81 +20,134 @@ import avatarImg from '../../Avatar/examples/avatarImg.svg';
+
 ```ts file="./DropdownGroups.tsx"
 ```
 
 ### Disabled
 
+Use the `isDisabled` property within the `DropdownToggle` component to disable the expansion of the entire dropdown menu.
+
 ```ts file="./DropdownDisabled.tsx"
 ```
 
+### Dropdown with descriptions
+
+Assign a `description` property and respective value to each `DropdownItem` that you wish you offer more context to. This description will appear in the menu itself, under the dropdown item it corresponds to.
+
+```ts file="./DropdownDescriptions.tsx"
+```
+
+### Dropdown with image and text
+
+Use the `icon` property to display an image in the `DropdownToggle`. Icons can be used alongside text, as shown in the following example.
+
+```ts file="./DropdownImageAndText.tsx"
+```
+
 ### Primary toggle
 
+You can adjust the styling of the `DropdownToggle` as needed. The following example uses the `isPrimary` to apply primary button styling. This method is deprecated and we now suggest using `toggleVariant="primary"` to acheive the same results.
+
 ```ts file="./DropdownPrimaryToggle.tsx"
 ```
 
 ### Secondary toggle
 
+This example uses `toggleVariant="secondary"` to apply secondary button styling to the `DropdownToggle`.
+
 ```ts file="./DropdownSecondaryToggle.tsx"
 ```
 
 ### Plain with text toggle
 
+The following example uses the `isText` and `isPlain`properties to plainly style the `Dropdown`.
+
 ```ts file="./DropdownPlainTextToggle.tsx"
 ```
 
 ### Position right
 
+The following example demonstrates a dropdown menu that is right aligned, which may be useful depending on the placement of your dropdown toggle and menu.
+
 ```ts file="./DropdownPositionRight.tsx"
 ```
 
 ### Alignments on different breakpoints
 
+The following example adjusts the alignment of the dropdown menu based on the size of the user's browser window.
+
 ```ts file="./DropdownAlignmentOnBreakpoints.tsx"
 ```
 
 ### Direction up
 
+The following example changes the direction of the menu from the default of downwards to an upwards direction instead. This may be useful depending on the placement of your dropdown toggle and menu.
+
 ```ts file="./DropdownDirectionUp.tsx"
 ```
 
 ### With kebab
 
+You may use a kebab icon instead of a text label.
+
 ```ts file="./DropdownKebab.tsx"
 ```
 
 ### With badge
 
+You may use a badge instead of a text label. A badge allows you to display the number of dropdown menu items.
+
 ```ts file="./DropdownBadge.tsx"
 ```
 
 ### Icon only
 
+You may use an icon instead of a text label.
+
 ```ts file="./DropdownIconOnly.tsx"
 ```
 
 ### Split button
 
+You may use a split button checkbox instead of a text label.
+
 ```ts file="./DropdownSplitButton.tsx"
 ```
 
 ### Split button (with text)
 
+A split button checkbox may be used in combination with a text label.
```

Comment:
I believe the intended use case for the split button with a checkbox is for bulk select operations on large data sets. Maybe that'd be a helpful thing to call out explicitly.